exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,6 @@
 const name = " Daniel  ";
 
+const message=`My name is Daniel and my name is 6 characters long`;
```

Comment:
Your code should be more versatile if you use variable instead of 'Daniel' and - name.length instead of '6'